src/app/collective-rewards/allocations/StakeHint.tsx

```diff
@@ -0,0 +1,11 @@
+import { Link } from '@/components/Link'
+
+export const StakeHint = () => (
+  <span>
+    Not enough stRIF to allocate. You can{' '}
+    <Link href={'/user'} className="text-[#E56B1A]" target="_blank">
+      Stake RIF
+    </Link>{' '}
+    to increase your balance
+  </span>
+)
```

Comment:
is this different from the voting power hint elsewhere in the app?

Reply:
I've just created a component for this because we use the same in 2 places (within the allocation page)